whylog/teacher/__init__.py

```diff
@@ -27,12 +83,16 @@ def update_pattern(self, line_id, proposed_pattern):
     def make_group(self, line_id, span):
         """
         Improves text pattern by adding group corresponding to param in text.
+
+        Removes (or maybe updates) constraints related to groups in line with line_id
         """
         pass
 
-    def remove_group(self, line_id, group_id):
+    def remove_group(self, pattern_group):
         """
         Improves text pattern by removing group corresponding to param in text.
+
+        Removes (or maybe updates) constraints related to groups in line with line_id
```

Comment:
The docs String identical to make_group function. I really don't understand why.
I neither understand what is pattern_group param.